home.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CC Global Summit</title>
+    <link rel="stylesheet" href="./home.css" />
+  </head>
+  <body>
+
+    <!-- Navigation Section Starts -->
+
+    <header>
+      <ul class="top-nav">
+        <li>
+          <img src="./images/facebook.png" alt="">
+        </li>
+        <li>
+          <img src="./images/twitter.png" alt="">
+        </li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+      <nav>
+        <div class="hamburger open" id="hamburger">&#9776;</div>
+        <a href="./home.html">
+          <img
+          class="nav-logo"
+          src="./images/logo.png"
+          alt="CC Global Summit Logo"
+          />
+        </a>
```

Comment:
- [ ] Good job on this, but please kindly ensure you stick to the design guidelines. If you look at the original design you will notice that the logo is not included in the navbar of the mobile screen.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/227737728-11ac9f95-2cb3-45da-9f80-6399f6de7a9b.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/227737701-ee11061c-efa5-4321-8514-18ac6c121d59.png)